packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -209,3 +210,29 @@ export class CCreateVault extends ComposableBuffer<CreateVault> {
     ]
   }
 }
+
+/**
+ * DepositToVault DeFi Transaction
+ */
+export interface DepositToVault {
+  vaultId: string // ---------------| 32 bytes, Vault Id
+  from: Script // ------------------| n = VarUInt{1-9 bytes}, + n bytes, Address containing collateral
+  amount: TokenBalanceVarInt
+}
+
+/**
+ * Composable DepositToVault, C stands for Composable.
+ * Immutable by design, bi-directional fromBuffer, toBuffer deep composer.
+ */
+export class CDepositToVault extends ComposableBuffer<DepositToVault> {
+  static OP_CODE = 0x53 // 'S'
+  static OP_NAME = 'OP_DEFI_TX_DEPOSIT_TO_VAULT'
+
+  composers (dv: DepositToVault): BufferComposer[] {
```

Comment:
DepositToVault = dtv right? Anyway, it is your call to if you decide to change this.